boards/xtensa/esp32/esp32-core/Kconfig

```diff
@@ -44,4 +44,23 @@ config ESP32CORE_FLASH_IMAGE
 	---help---
 		Create flash_image.bin mainly used for QEMU.
 
+choice
```

Comment:
The SPIFLASH should work independently from the File System the user chose, maybe it is a limitation of current implementation, correct?

Reply:
We could add any other file system, I just picked some of what we have in the repo.
The board logic that was added with this PR prepares the driver to be mounted on a file system, but the SPIFLASH driver itself is completely independent from that.